addons/trenches/functions/fnc_automaticFilePath.sqf

```diff
@@ -0,0 +1,48 @@
+#include "script_component.hpp"
+/*
+ * Author: Salbei
+ * Automatic creation of a path to surface texture.
```

Comment:
What's the purpose of this function? What does this mean? Why does it use publicVariable at the end?

Reply:
If activated it will try to create a path for the ground texture file based of configs provided by the terrain, only works with terrains that are not an island.

Just trying to expand the amount the trenches cover